app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Work history iteration
+description: Moving the work history within Safeguarding section
+date: 2021-01-08
+related:
+  items:
+  - text: Work history breaks
+    href: https://bat-design-history.netlify.app/apply-for-teacher-training/work-history-breaks/
+    description: Previous post about breaks in work history
+  - text: Work history - August iteration
+    href: https://bat-design-history.netlify.app/apply-for-teacher-training/work-history/
+    description: Previous change to the work history flow
+---
+Discussions with providers revealed that many of them require a “full” work history, rather than just work within the past 5 years, for safeguarding reasons.
+
+Candidates were spending a significant amount of time filling in the “Skills and experience relevant to teaching” question for each job, treating these as a “mini personal statement”. However providers indicated that they didn’t always read these in detail.
+
+Analysis of validation errors also revealed that candidates were often leaving the “Skills and experience relevant to teaching” question blank, which then displays an error. Our hypothesis was because they wanted to come back and write this section later.
+
+## What we did
+
+The work history section was moved beneath the “Safeguarding” heading on the application review page. This was to try and communicate to candidates that the purpose of the work history was mainly for safeguarding, rather than something used to direct assess them.
+
+We changed the “How long have you been working?” question to “Do you have any work history?” as we no longer needed to guide candidates they only needed to enter their past 5 years of work history.
+
+An answer to this question was added of “No, I have always been in full time education”. This was based on analysis showing that around 20% of candidates are 20 or 21 years old, and have likely gone straight from A-Levels to University, which may explain their lack of work history. Adding this option avoids the need to them to have to explain this in their own words.
+
+If someone selected “No, I do not have any work history”, then we revealed a textbox asking “Tell us why you’ve been out of the workplace” with some hint text which gave three examples. These were very brief, to indicate to candidates that they did not need to be super detailed.
+
+When someone indicated they did have work history, we took them to a page with an empty table and some guidance that they needed to add their full work history. They could then add a first job.
```

Comment:
......._an empty table and guidance to add their full work history. They could then add a first job._